collector/index.mdx

```diff
@@ -29,3 +29,33 @@ The collector makes only outbound connections, it does not listen on any ports.
 Heroku-based setup, where the collector listens on the dyno's PORT for Heroku to consider the application running,
 as well as to receive database logs by serving as a Heroku log drain.
 
+## Data collection and Snapshot submission Frequency
+
+The pganalyze collector gathers metrics and metadata through various snapshots, each with specific cadences.
+This section explains the types of data collected and when different snapshot types are sent.
+
+There are three different types of snapshots that are sent to the pganalyze app.
+
+* **Compact Activity Snapshot**: Sent every 10 seconds
+* **Compact Logs Snapshot**:
+  * Streaming: Sent approximately every 10 seconds
+  * Download: Sent every 30 seconds
```

Comment:
If we're breaking this out, maybe we should explain when which kind of log collection happens...

Reply:
Yeah, I agree on this, but honestly I haven't figured out the difference between them 🙃 I'm planning to do some follow-up around this area to explain better once I figured it out 👍